ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -955,8 +955,8 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-workers-marketplace
-- as: aws-ocm-osd-ccs-f7
-  cron: 2 12 2,9,16,23 * *
+- as: aws-ocm-osd-ccs-f14
+  cron: 36 19 14,29 * *
```

Comment:
Setting the 29th in crontab might not be executed in February.

Reply:
Good catch.